frontend/src/layout/navigation-3000/sidebars/insights.ts

```diff
@@ -136,6 +136,13 @@ export const insightsSidebarLogic = kea<insightsSidebarLogicType>([
         // kea-typegen doesn't like selectors without deps, so searchTerm is just for appearances
         debounceSearch: [(s) => [s.searchTerm], () => true],
     })),
+    listeners(({ values, cache }) => ({
+        loadInsights: () => {
+            if (!values.paramsFromFilters.offset) {
+                cache.requestedInsights.length = 0
```

Comment:
🤔 is there anything that prevents a reset like this:

```suggestion
                cache.requestedInsights = []
```